sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -161,9 +166,18 @@ func App() *cli.App {
 					Destination: &useCachedImage,
 					Value:       false,
 				},
+				&cli.StringSliceFlag{
+					Name:  "label",
+					Usage: "add multiple labels to the pulled image. The label should be in the format of `key=value`",
```

Comment:
nit: could condense this a bit:
```suggestion
					Usage: "label to add to the pulled image in `key=value` format",
```